src/navigations/HomeBottomTabs.js

```diff
@@ -46,6 +48,11 @@ function HomeBottomTabs() {
     }
   }, [initialStartup, otherProfileData]);
 
+  React.useEffect(() => {
+    const totalUnreadCount = (signedChannelUnreadCount || 0) + (anonymousChannelUnreadCount || 0);
+    udpateAppBadgeWith(totalUnreadCount);
+  }, [signedChannelUnreadCount, anonymousChannelUnreadCount]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This new effect updates the app badge with the total unread count whenever the signed or anonymous channel unread counts change. This looks good, but please ensure that the `updateAppBadgeWith` function handles all edge cases correctly, such as when the unread counts are undefined or not numbers.



<!-- This is an auto-generated comment by OSS CodeRabbit -->